Add performance considerations (#16)
tlswg/draft-ietf-tls-ticketrequest: draft-ietf-tls-ticketrequests.md

for ExtensionType (defined in {{RFC8446}}), with "TLS 1.3" column values being set to
"CH", and "Recommended" column being set to "Yes".

# Performance Considerations

> Servers can send the requested tickets in NewSessionTicket messages at any time after the

Reviewer: ```suggestion
Servers can send tickets in NewSessionTicket messages any time after the
```

> client Finished message. If the server chooses to send a large number of tickets to the client,

Reviewer: ```suggestion
server Finished message (see {{RFC8446}}; Section 4.6.1). A server which chooses to send a large number of tickets to a client
```

Author: That section say client finished though... 

`At any time after the server has received the client Finished message, it MAY send a NewSessionTicket message.`

Reviewer: It also says this:

```
Note: Although the resumption master secret depends on the client's
   second flight, a server which does not request client authentication
   MAY compute the remainder of the transcript independently and then
   send a NewSessionTicket immediately upon sending its Finished rather
   than waiting for the client Finished.
```

Author: Ahhh I see

> it can potentially harm application performance if the tickets are sent before application data.

Reviewer: ```suggestion
can potentially harm application performance if the tickets are sent before application data.
```

Author: I don't think removing the word works, grammatically?

Reviewer: If you apply them all it should :)

Author: Ah the other part had rolled off

> For example, if the transport connection has a constrained congestion window, the ticket

Reviewer: ```suggestion
For example, if the transport connection has a constrained congestion window, ticket
```

> messages could delay the sending of application data. To avoid this, servers should

Reviewer: ```suggestion
messages could delay sending application data. To avoid this, servers should
```
